src/libs/browser-storage.js

```diff
@@ -76,8 +100,11 @@ export const listenDynamic = (storage, key, fn) => {
 /**
  * Returns a stateful object that manages the given storage location.
  * Implements the Store interface, and can be passed to useStore.
+ *
+ * Note that this method will use static storage, meaning that it will not
+ * automatically be deleted in the case of space overflow. If the data you are using
+ * is transient, consider using `setDynamic` instead.
```

Comment:
Should we add `@param` here for the param documentation to be consistent? 
Also, I would suggest moving line 101 to the end and annotating with `@returns`
Perhaps something like:
```
@returns A stateful object that manages the given storage location.
@see atom for more {@link https://github.com/DataBiosphere/terra-ui/blob/999656a2cb159d73f0342130ccdb37215bb171c3/src/libs/utils.js#L32}
```
I'm not sure how folks feel about adding links to other parts of the code. We could leave that out.

Reply:
I added the `params` and the `returns`, but didn't do the `see` because I didn't want the file to need the atom import. I don't like putting in a URL link because it is sure to get out-of-date.